app/models/sidebar_url.rb

```diff
@@ -5,6 +5,71 @@ class SidebarUrl < ActiveRecord::Base
   MAX_ICON_LENGTH = 40
   MAX_NAME_LENGTH = 80
   MAX_VALUE_LENGTH = 200
+  COMMUNITY_SECTION_LINKS = [
+    {
+      id: -1,
+      name: I18n.t("sidebar.sections.community.links.everything.content", default: "Everything"),
```

Comment:
This would default the links to the site's default locale which breaks the interface for users that have selected another locale.

Reply:
You are right, this is a tricky part and I didn't find a better approach. At this stage, the content of the link is still safe, as it is hard-coded in the specific link:

https://github.com/discourse/discourse/blob/main/app/assets/javascripts/discourse/app/lib/sidebar/common/community-section/everything-section-link.js#L47

I wanted to set a reasonable value for the next step – edit form. It is possible that admin will have to translate links to another language. 

Tricky part is that we are switching from I18n system to edit and put whatever name admin wants mode.

Reply:
Do we allow an admin to change the name of the links? I don't think that should be allowed because the name of those links have been carefully chosen for the best UX experience for users.

Reply:
I think we should just hardcore the name of the link in english for now and use the name on the client side to interpolate for the translation. If the translation doesn't exist on the client side, then we default to the name. Otherwise, we will end up with "everything" links with different locale names in the DB which may make it hard to query for in the future.